src/main/lua/exasolrls/rls_adapter.lua

```diff
@@ -29,10 +30,7 @@ end
 -- @return response containing the metadata for the virtual schema like table and column structure
 --
 function M.create_virtual_schema(_, request)
-    local properties = request.schemaMetadataInfo.properties or {}
-    validate(properties)
-    local schema_metadata = metadata_reader.read(properties.SCHEMA_NAME)
-    return {type = "createVirtualSchema", schemaMetadata = schema_metadata}
+    return {type = "createVirtualSchema", schemaMetadata = handle_schema_scanning_request(request)}
```

Comment:
```suggestion
    return {type = "createVirtualSchema", schemaMetadata = get_schema_metadata(request)}
```
❓

Reply:
Strictly speaking the method does two things: extract properties and get the metadata. You could argue that it breaks the separation of concerns. But I wanted to eliminate the code duplication. That's why I chose that name.

Especially since I expect I will use the exact same code again in `setProperties` in another ticket, I want reuse.